cvmfs/util/posix.cc

```diff
@@ -1808,15 +1883,8 @@ bool ManagedExec(const std::vector<std::string>  &command_line,
     }
 
     // Child, close file descriptors
-    max_fd = static_cast<int>(sysconf(_SC_OPEN_MAX));
-    if (max_fd < 0) {
-      failed = ForkFailures::kFailGetMaxFd;
-      goto fork_failure;
-    }
-    for (int fd = 0; fd < max_fd; fd++) {
-      if ((fd != pipe_fork.write_end) && (preserve_fildes.count(fd) == 0)) {
-        close(fd);
-      }
+    if (!CloseAllFildes(skip_fds)) {
+      failed = ForkFailures::kUnknown;
```

Comment:
```suggestion
      failed = ForkFailures::kFailGetMaxFd;
      goto fork_failure;
```

Reply:
Yes, I wasn't very happy with `kUnknown` either. Just to clarify, do you want me to drop the rename commit and use `kFailGetMaxFd` instead?

Reply:
Eh, sorry, my bad.  I meant the new kFailCloseFds of course.

Reply:
I updated the suggestion

Reply:
I somehow missed renaming this bit and actually using the new error code, sorry. It should be fixed now.